pinax/blog/views.py

```diff
@@ -164,6 +183,7 @@ def blog_feed(request, section=None, feed_type=None):
     else:
         raise Http404()
 
+    # @@@ how do we handle scoped kwarg here
```

Comment:
Ah, ha! My point earlier in the `Post` model was true and you noticed it too. To handle scoping here you need to reuse the code in `Post.scoping_url_kwargs`. Once that code is reusable outside of `Post` you can use the returned dictionary in `reverse("pinax_blog:blog")` below.